src/Tizen.Telephony/Tizen.Telephony/Modem.cs

```diff
@@ -80,8 +80,14 @@ public enum PowerStatus
         /// Gets the IMEI (International Mobile Station Equipment Identity) of a mobile phone.
         /// The IMEI number is used by a GSM network to identify valid devices and therefore, can be used for stopping a stolen phone from accessing that network.
         /// </summary>
+        /// <remarks>
+        /// For avoding the unexpected behavior of old version applications that have http://tizen.org/privilege/telephony privilege. There is an exceptional handling in case of permission denied.
```

Comment:
Spelling, should be `avoiding`. It is correct, but I suggest using `To avoid` instead. It should be one sentence.

Reply:
Done